### PR TITLE
add variables structure to appearance, update variable names

### PR DESCRIPTION
**Pull request by bkirk-stripe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/connect-js",
-  "version": "1.0.9",
+  "version": "1.0.10",
```
Comment (jorgea-stripe) [inline package.json:3]: You shouldn't bump this in this PR! We should bump after the regular change is merged. The PR should also have a reasonable description/title.

File: types/shared.d.ts [hunk 2]
```diff
@@ -63,6 +63,9 @@ export declare type AppearanceOptions = {
   badgeBorderRadius?: string;
   overlayBorderRadius?: string;
 
+  // Overlay
+  overlayZIndex?: string;
```
Comment (jorgea-stripe) [inline types/shared.d.ts:67]: Can we close on the string vs. number topic before merging this one?

**Review by jorgea-stripe — COMMENTED** [review]
Some feedback!

File: types/shared.d.ts (hunk 2 above)
Comment (bkirk-stripe) [reply types/shared.d.ts]: closed on this - we are going to expose it as a number

**Review by mxl-stripe — APPROVED** [review]

**Review by jojo-stripe — APPROVED** [review]

**Pull request merged**